_posts/2021-02-11-Creating-A-Game.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Creating A Game
+description: 
+summary: 
+tags:
+---
+Today, I did typing club for about an hour. We also built a game, we had to create a new folder locally and work through a tutorial that Kajari provided for us . [This is the article we used](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript). 
+After we were done with the game, we had to push up the game up to a repository in Github. For this we used [this article](https://docs.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line). To end this we had to turn the game into a Github page and add the link onto our Github account.
+After lunch, I met with Kajari and she provided some feedback. One thing that I need to pay more attention to is to small details, I think that I tend to get overwhelmed and I stress about finishing something, so I tend to rush things. I will be working on paying more attention to details and learning how to take things calmly.
```

Comment:
No need to talk about the feedback that I gave you :) Except for the writing feedback -- particularly that you should set the context clearly and take notes for your future self so that you can figure out how you did this work and when you are writing your own game later on you can look up these links to figure out how you did it and can repeat/build on it.